src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1368,6 +1381,7 @@ class ApplicationManagerImpl
   sync_primitives::Lock audio_pass_thru_lock_;
   sync_primitives::Lock tts_global_properties_app_list_lock_;
   hmi_apis::Common_DriverDistractionState::eType driver_distraction_state_;
+  bool lock_screen_dismissal_enabled_state_;
```

Comment:
@IGapchuk word `state` is redundant

Reply:
@LuxoftAKutsan Fixed in bd9042870da8f0a7750f1b940b25411e77769044.